test/test_parameters.jl

```diff
@@ -0,0 +1,6 @@
+@testset "Parameters" verbose=true begin
```

Comment:
I think we also want to do this tests for all the different types not only `Real`